src/DocumentationBrowserViewExtension/DocumentationBrowserViewExtension.cs

```diff
@@ -251,7 +251,14 @@ private void GroupInsertedGraph(List<AnnotationViewModel> existingGroups, string
                 var annotationViewModel = DynamoViewModel.CurrentSpaceViewModel.Annotations
                         .First(x => x.AnnotationModel == annotation);
 
-                var styleItem = annotationViewModel.GroupStyleList.First(x => x.Name.Equals(DynamoProperties.Resources.GroupStyleDefaultReview));
+                GroupStyleItem styleItem = null;
```

Comment:
This change is already merged into master, right?

Reply:
oh yeah, that's strange the cherrypick included here should be there already